src/components/stepforms/widgets/InputDate.vue

```diff
@@ -0,0 +1,143 @@
+<template>
+  <div class="widget-input-date__container" :class="toggleClassErrorWarning">
+    <div class="widget-input-date__container">
+      <label v-if="name" @click="$refs.input.focus()">{{ name }}</label>
+      <a v-if="docUrl" :href="docUrl" target="_blank" rel="noopener">
+        <i class="fas fa-question-circle" aria-hidden="true" />
+      </a>
+    </div>
+    <VariableInput
+      :value="value"
+      :available-variables="availableVariables"
+      :variable-delimiters="variableDelimiters"
+      :has-arrow="true"
+      @input="updateValue"
+    >
+      <input
+        ref="input"
+        :class="elementClass"
+        :placeholder="placeholder"
+        type="date"
```

Comment:
For now I think it's fine but if we have also times (hours, minutes, seconds) we will need to add an extra <input type="time"> 

(Mos cases are just dates so it seems OK to me not to take this into account